README.md

```diff
@@ -2,46 +2,61 @@
+Have you ever activated a WordPress plugin that has a conflict with another because the plugins use two different versions of the same PHP library? **Strauss is the solution to that problem** - it ensures that _your_ plugin's PHP dependencies are isolated and loaded from your plugin rather than loading from whichever plugin's autoloader registers & runs first.
 
 > ⚠️ **Sponsorship**: I don't want your money. [Please write a unit test to help the project](https://brianhenryie.github.io/strauss/).
 
-## Breaking Changes
-
-* v0.16.0 – will no longer prefix PHP built-in classes seen in polyfill packages
-* v0.14.0 – `psr/*` packages no longer excluded by default
-* v0.12.0 – default output `target_directory` changes from `strauss` to `vendor-prefixed`
+## Table of Contents
 
-Please open issues to suggest possible breaking changes. I think we can probably move to 1.0.0 soon. 
+* [Installation](#installation)
+    * [As a `.phar` file](#as-a-phar-file-within-composerjson-recommended) (recommended)
+    * [As a dev dependency via composer](#as-a-dev-dependency-via-composer-not-recommended)  (not recommended)
+    * [Adding `composer dump-autoload`](#adding-composer-dump-autoload)
+* [Usage](#usage)
+* [Configuration](#configuration)
+* [Autoloading](#autoloading)
+* [Motivation & Comparison to Mozart](#motivation--comparison-to-mozart)
+* [Alternatives](#alternatives)
+* [Breaking Changes](#breaking-changes)
+* [Acknowledgements](#acknowledgements)
 
-## Use
+## Installation
 
-Require as normal with Composer:
+### As a `.phar` file (recommended)
 
-`composer require --dev brianhenryie/strauss`
+There are a couple of small steps to make this possible.
 
-and use `vendor/bin/strauss` to execute.
+#### Create a `bin/.gitkeep` file
 
-Or, download `strauss.phar` from [releases](https://github.com/BrianHenryIE/strauss/releases), 
+This ensures that there is a `bin/` directory in the root of your project. This is where the `.phar` file will go.
 
-```shell
-curl -o strauss.phar -L -C - https://github.com/BrianHenryIE/strauss/releases/latest/download/strauss.phar
+```bash
+mkdir bin
+touch bin/.gitkeep
 ```
 
-Then run it from the root of your project folder using `php strauss.phar`. 
+#### `.gitignore` the `.phar` file
 
-To update the files that call the prefixed classes, you can use `--updateCallSites=true` which uses your autoload key, or `--updateCallSites=includes,templates` to explicitly specify the files and directories.
+Add the following to your `.gitignore`:
 
-Its use should be automated in Composer scripts.
+```bash
+bin/strauss.phar
+```
+
+#### Edit `composer.json` `scripts
+
+In your `composer.json`, add `strauss` to the `scripts` section:
 
 ```json
 "scripts": {
     "prefix-namespaces": [
-        "strauss"
+        "sh -c 'test -f ./bin/strauss.phar || curl -o bin/strauss.phar -L -C - https://github.com/BrianHenryIE/strauss/releases/latest/download/strauss.phar'",
```

Comment:
Why is `sh -c` needed? I think that should work anyway. Maybe for GitHub Actions?

Reply:
Without sh -c, the initial run when the phar file does not exist downloads and errors out the command. Wrapping in the sh -c allows it to do the download in isolation and then continue on with the composer command without error. 

Perhaps there’s a cleaner way, but that is what I use!